test_scripts/Polices/build_options/022_ATF_P_TC_PTU_Merge_Into_Local_PT_HTTP.lua

```diff
@@ -184,40 +151,55 @@ function Test:TestStep_RegisterNewApp()
   local corId = self.mobileSession2:SendRPC("RegisterAppInterface", config.application2.registerAppInterfaceParams)
   self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
 
-  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
-  EXPECT_HMICALL("BasicCommunication.PolicyUpdate", {file = "/tmp/fs/mp/images/ivsu_cache/sdl_snapshot.json"})
-  :Do(function(_,data)
-      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"}, {status = "UPDATING"}):Times(2)
+
+  self.mobileSession2:ExpectNotification("OnSystemRequest", {status = "LOCK_SCREEN_ICON_URL"})
+
+  self.mobileSession:ExpectNotification("OnSystemRequest")
+  :Do( function(_, data)
+      print("Mob1: OnSystemRequest, requestType: "..data.payload.requestType)
+      if(data.payload.requestType == "HTTP") then
+        ptu = json.decode(data.binaryData)
```

Comment:
@istoimenova, Please add check on empty ptu since if snapshot is not provided as binary data it could be empty (blank)